### DISCUSSION TITLE
FluentCard will cut off select box lists

### DISCUSSION THREAD

**Discussion by StevenTCramer** [microsoft/fluentui-blazor#2149] [Q&A]
If you have a FluentSelect inside a FluentCard the list will be cut off at the border of the card.

The reason is because the card has `contain: content;` 

I did a search and saw where this was added.

https://github.com/microsoft/fluentui-blazor/commit/1006573e1dcdd80d16bb14d092d7b9fef1432388

There is a comment:

"* Fix style adding `contain: content;`"

But I don't know exactly what "fix" means.  

## Workaround

One can work around this by setting the contain to none.

```razor
<style>
  @($@"
    #{Id} {{
      contain: none;
    }}
  ")
</style>
```

**Comment by dvoituron** [comment]
Have you tried adding `AreaRestricted="false"` as explained on this page? 
https://www.fluentui-blazor.net/Card

**Reply by StevenTCramer** [reply]
I have but it still doesn't work.

contain:contnet is still applied.

![image](https://github.com/microsoft/fluentui-blazor/assets/357219/c4cce532-67db-48aa-a7b6-93948a8b4b0f)

**Reply by StevenTCramer** [reply]
I see the issue... 

```razor
<FluentCard Id=@Id MinimalStyle=true AreaRestricted=false>
```

With `MinimalStyle=true`, `AreaRestricted` does NOT work.

```razor
<FluentCard Id=@Id MinimalStyle=false AreaRestricted=false>
```
With `MinimalStyle=false`, `AreaRestricted` does work.

**Reply by dvoituron** [reply]
Using `AreaRestricted="false"` a style is automatically added to set `contain: style;` 

![image](https://github.com/microsoft/fluentui-blazor/assets/8350694/5054b94d-c2a5-4cd3-99d1-32cf8f010172)

```csharp
protected string? StyleValue => new StyleBuilder(Style)
    .AddStyle("--card-width", Width, !string.IsNullOrEmpty(Width))
    .AddStyle("--card-height", Height, !string.IsNullOrEmpty(Height))
    .AddStyle("content-visibility", "visible", !AreaRestricted && !MinimalStyle)
    .AddStyle("contain", "style", !AreaRestricted && !MinimalStyle)
    .Build();
```

What is you issue with that?

**Reply by StevenTCramer** [reply]
I will do a PR when I have a min. And add test cases.

**Reply by dvoituron** [reply]
> I will do a PR when I have a min. And add test cases.

Why? You need to use only `AreaRestricted="false"` not `MinimalStyle="true"`
